server/scripts/import-tests/index.js

```diff
@@ -15,6 +15,10 @@ const {
     getTestPlans,
```

Comment:
Is there a specific section to point out? The majority of the v1 vs v2 complexity is now included in `func:createTestsForFormat` following Stalgia's comment.

Some of the existing functions were renamed and to extend support of what they were originally doing. But I can add additional clarifying comments which attributes have been added.

The reason that binary logic can't happen further up is because of when the v2 evaluation for an example's folder has to happen.